ExoCTK/contam_visibility/visibilityPA.py

```diff
@@ -15,7 +15,6 @@
 import math
 import numpy as np
 import matplotlib
-matplotlib.use('Agg')

```

Comment:
Like I mentioned -- we need this to be able to write `matplotlib` plots on the server.

Reply:
Sure. I'll put it back in.